tableaudocumentapi/connection.py

```diff
@@ -111,3 +166,12 @@ def authentication(self):
     @property
     def dbclass(self):
         return self._class
+
+    @dbclass.setter
+    def dbclass(self, value):
+
+        if value not in KNOWN_DB_CLASSES:
```

Comment:
Rather than making the assumption that KNOWN_DB_CLASSES is a list, create a method called "isValidDbClass(value)" in case in the future we decide to be smarter about how we determine if it's a valid db class.